packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -412,5 +412,20 @@ export default class PostsRepo extends AbstractRepo<DbPost> {
       WHERE contents->>'html' LIKE '%elicit-binary-prediction%'
     `);
   }
+
+  async getStaleDialogues(): Promise<DbPost[]> {
+    const tenDaysAgo = new Date(Date.now() - (10 * 24 * 60 * 60 * 1000));
+    return this.getRawDb().any(`
+      -- PostsRepo.getStaleDialogues
+      SELECT p.*
+      FROM "Posts" p
+      JOIN "Revisions" r ON p."_id" = r."documentId"
+      WHERE
+        p."draft" = TRUE AND
+        p."collabEditorDialogue" = TRUE AND
+        r."editedAt" < $1
+      GROUP BY p."_id"
```

Comment:
It took me a bit of time to figure out what this query was trying to do (though it does seem to do it correctly, from comparing it to a version using a subquery instead of a join). 

Also it seems kind of expensive... if it's only running once a day that's probably fine, but I'd add a docstring being quite explicit about the intended use-case.